tests/hypervisor/conftest.py

```diff
@@ -1,6 +1,8 @@
 import pytest
 from virtwho import RHEL_COMPOSE
 
+server_xxx_error = ["Name or service not known", "No address associated with hostname"]
+
```

Comment:
In my opinion, if this is just for esx hypervisor, it is better to include it in the `esx_assertion()` function, and maybe re-name to ``server_invalid_error`` or sth. like this would be great :)

Reply:
renamed it.
but because it is called in the esx, hyperv and libvirt assertion, so take it out of the functions.

Reply:
oh, my bad